src/hooks/useDrag.tsx

```diff
@@ -478,7 +595,8 @@ export const onCreate = <E,>(
 export const onDelete = (
   homepageState: EditorHomepageState,
   elemType: EditorHomepageElement,
-  indexToDelete: number
+  indexToDelete: number,
+  subindexToDelete = 0
```

Comment:
i might be missing something here - could you point me to why we are adding a new argument here? i know it's used down below but wouldn't it be possible to pass in `subelemType` + `subIndex`?

Reply:
See above!